var/spack/repos/builtin/packages/cosma/package.py

```diff
@@ -35,7 +35,9 @@ class Cosma(CMakePackage):
     version("2.0.7", sha256="8d70bfcbda6239b6a8fbeaca138790bbe58c0c3aa576879480d2632d4936cf7e")
     version("2.0.2", sha256="4f3354828bc718f3eef2f0098c3bdca3499297497a220da32db1acd57920c68d")
 
+    depends_on("c", type="build")
     depends_on("cxx", type="build")  # generated
+    depends_on("fortran", type="build")
```

Comment:
```suggestion
```

I think the project is written entirely in C++, the C/Fortran interface is generated with a C++ compiler too.

It does require a C compiler but I believe it's redundant and could be deleted upstream https://github.com/eth-cscs/COSMA/blob/3c0b2129dfcfed67c7b30d388734aac58009a7f4/CMakeLists.txt#L7.

Unless I'm missing something only c/cxx are required in Spack

Reply:
Even after the update of the [CMakeLists.txt file](https://github.com/eth-cscs/COSMA/blob/master/CMakeLists.txt) upstream, a Fortran compiler is still requested by `cmake` for the build of `cosma@master` with `spack`.